lib/workers/jobs.js

```diff
@@ -142,7 +142,7 @@ jobs.signInput = function signInput(tx, index, coin, ring, type) {
  */
 
 jobs.ecVerify = function ecVerify(msg, sig, key) {
```

Comment:
No need to change the name of this function?

Reply:
No, its still elliptic curve verify